doc/guide/start.rst

```diff
@@ -0,0 +1,453 @@
+  $ flux resource info
+  2 Nodes, 8 Cores, 0 GPUs
+  $ flux run --label-io -N2 hostname
+  0: fluke6
+  1: fluke7
+  $ exit
+  [detached: session exiting]
+  $
+
+Now let's try batch:
+
+.. code-block:: console
+
+  $ flux batch -N2 ./workload.sh
+  f2SoPsfdA7Ub
+  $ cat flux-f2SoPsfdA7Ub.out
+   10:25:05 run 4.8s,  owner alice,  depth 1,  size 2
+  2 Nodes, 8 Cores, 0 GPUs
+  0: fluke6
+  1: fluke7
+
+Like the allocation example, the :man1:`flux-batch` command spawns a new Flux
+instance that runs for the duration of its initial program, the batch script
+``workload.sh``.  The Flux design does not require job steps or a specialized
+step scheduler because of the recursive launch property.
+
+Although :man1:`flux-batch` has other nice features like support for batch
+directives and copying of the batch script at submission, a similar result
+can be obtained for this simple example with :man1:`flux-submit` and
+:man1:`flux-start`:
+
+.. code-block:: console
+
+  $ flux submit --output=flux-{{id}}.out -N2 flux start ./workload.sh
+  f2SoiFq3N5Jo
+  $ cat flux-f2SoiFq3N5Jo.out
+   11:05:24 run 4.7s,  owner alice,  depth 1,  size 2
+  2 Nodes, 8 Cores, 0 GPUs
+  0: fluke6
+  1: fluke7
+
+Starting with Hydra
+-------------------
+
+MPICH `hydra <https://github.com/pmodels/mpich/blob/main/doc/wiki/how_to/Using_the_Hydra_Process_Manager.md>`_
+is an MPI launcher with PMI support that can start a Flux instance across
+multiple nodes using ssh.  It is sometimes just the right tool needed to get
+a Flux instance going when nothing else will work.
+
+Given a ``hosts`` file containing:
+
+::
+
+  test0
+  test1
+  test2
+  test3
+  test4
+  test5
+  test6
+  test7
+
+And assuming passwordless ssh works for those nodes, one can run:
+
+.. code-block:: console
+
+  $ mpiexec.hydra -launcher ssh -f hosts flux start ./workload.sh
+   23:23:14 run 3.9s,  owner alice,  depth 0,  size 8
+  8 Nodes, 32 Cores, 0 GPUs
+  0: test0
+  2: test2
+  1: test1
+  5: test5
+  6: test6
+  7: test7
+  4: test4
+  3: test3
+
+Starting with Systemd
+---------------------
+
+A Flux system instance can be started by `systemd <https://systemd.io/>`_.
+In this case, the PMI bootstrap phase of instance startup in which Flux
+brokers determine their rank and network endpoints dynamically is replaced
+with static configuration, and broker startup and shutdown follows the normal
+systemd service pattern.
+
+The systemd unit file starts the broker as the ``flux`` user and sets a few
+broker options such as:
+
+- The broker ``rundir`` is set to ``/run/flux``
+
+- The broker ``statedir`` is set to ``/var/lib/flux``.  When the broker is
+  restarted, it saves/restores its state to this location.
+
+- The configuration directory is set to ``/etc/flux/system/conf.d``.
+
+- The local API socket is set to ``/run/flux/local``
+
+- The rank 0 broker is set become usable before all the other brokers have
+  joined, and to tolerate brokers joining and leaving as nodes go up and down.
```

Comment:
english error in first sentence, not sure what was intent.  "The rank 0 broker must be usable before all other ..., so flux can tolerate ..."

Reply:
I meant "is set to become usable".  Will fix.